ext/js/pages/settings/dictionary-controller.js

```diff
@@ -463,6 +544,19 @@ export class DictionaryController {
         modal.setVisible(true);
     }
 
+    /**
+     * @param {string} dictionaryTitle
+     */
+    updateDictionary(dictionaryTitle) {
```

Comment:
I honestly feel a bit weird about keying dictionaries by title? It feels brittle if dictionaries titles change. Not sure how often it happens and whether we already have strong precedent for this, but this is normally a code smell

Reply:
Unfortunately yes.... we already write dictionaries to the database using the title as the key.

Reply:
We enforce unique names on import, and we already use them as ID for single dict handlebars and when scoping dict CSS. I think it's fine to use as a key.